.github/workflows/actions.yml

```diff
@@ -43,9 +101,15 @@ jobs:
     - name: Install tox
       run: |
         python -m pip install --upgrade tox
+    - name: Get Tox Environment Name From Matrix Name
```

Comment:
I do not fancy depending on other repositories for a single liner. I know is should be safe due to due use of tags but still prefer to have less dependencies.

I found more practical to do a little bit of duplication and add an explicit `tox_env` variable defined in each matrix element. It adds few lines but makes is much easier to maintain.